src/Compilers/CSharp/Portable/Symbols/Source/SourceLocalSymbol.cs

```diff
@@ -597,40 +603,35 @@ private class PossiblyImplicitlyTypedOutVarLocalSymbol : SourceLocalSymbol
                                  binder.ScopeDesignator == declarationExpression.Parent.Parent :
                                  binder.ScopeDesignator.Contains(declarationExpression.Parent.Parent.Parent));
 #endif
+                _containingInvocation = containingInvocation;
             }
 
+            internal override SyntaxNode ForbiddenZone => _containingInvocation;
+
+            internal override ErrorCode ForbiddenDiagnostic => ErrorCode.ERR_ImplicitlyTypedOutVariableUsedInTheSameArgumentList;
```

Comment:
No, because in code such as `M(out var x, F(out x))` such a message would be less than illuminating.

---
In reply to: [73917941](https://github.com/dotnet/roslyn/pull/12991#discussion_r73917941) [](ancestors = 73917941)